client/lib/user/shared-utils/initialize-current-user.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from '@automattic/calypso-config';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isSupportUserSession,
+	isSupportNextSession,
+	supportUserBoot,
+	supportNextBoot,
+} from 'calypso/lib/user/support-user-interop';
+import { filterUserObject, rawCurrentUserFetch } from './';
```

Comment:
See above.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/54382/commits/5d63e9e3fceb965ad097aab4fff8f1f2413f1b23